client/lib/abtest/active-tests.js

```diff
@@ -37,8 +37,8 @@ module.exports = {
 	signupStepOneCopyChanges: {
```

Comment:
@vindl Wouldn't it be better to keep the test in there so that it's obvious why it's there? I worry that by adding this check it make it harder to track down all occurrences of the test. thoughts?